aptos-move/aptos-abstract-gas-usage/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "aptos-abstract-gas-usage"
 version = "0.1.0"
 edition = "2021"
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+license = { workspace = true }
```

Comment:
Silly question: why do we need to add this entry to some toml files, but not others? 😄

Reply:
I believe all the other ones have `license` already and just these ones were missing it and therefore unlicensed according to the tool.